src/torchcodec/_core/AVIOBytesContext.cpp

```diff
@@ -13,7 +13,7 @@ AVIOBytesContext::AVIOBytesContext(const void* data, int64_t dataSize)
     : dataContext_{static_cast<const uint8_t*>(data), dataSize, 0} {
   TORCH_CHECK(data != nullptr, "Video data buffer cannot be nullptr!");
   TORCH_CHECK(dataSize > 0, "Video data size must be positive");
-  createAVIOContext(&read, &seek, &dataContext_);
+  createAVIOContext(&read, nullptr, &seek, &dataContext_);
```

Comment:
Below: created a new `AVIOToTensorContext` class instead of using `AVIOBytesContext`. The constructor is different, and so is the underlying `DataContext` struct.

I'm happy to consider alternative solutions.